lte/gateway/c/oai/tasks/amf/Registration.cpp

```diff
@@ -596,10 +596,9 @@ int amf_send_registration_accept(amf_context_t* amf_context) {
        * Start T3550 timer
        */
       OAILOG_INFO(LOG_AMF_APP, "Timer: registration_accept timer start\n");
-      registration_proc->T3550.id = start_timer(
-          &amf_app_task_zmq_ctx, REGISTRATION_ACCEPT_TIMER_EXPIRY_MSECS,
-          TIMER_REPEAT_ONCE, registration_accept_t3550_handler,
-          (void*) registration_proc->ue_id);
+      registration_proc->T3550.id = amf_app_start_timer(
+          REGISTRATION_ACCEPT_TIMER_EXPIRY_MSECS, TIMER_REPEAT_ONCE,
+          registration_accept_t3550_handler, registration_proc->ue_id);
```

Comment:
do we need 2 log statements? timer start, timer started. log statements cause I/O overhead on the network function. let us takeout redundant log statement.